src/components/library/LibrarySearch.jsx

```diff
@@ -200,7 +200,14 @@ const LibrarySearch = () => {
         </div>
         <div className='Library-SearchBar-Wrapper'>
           <label htmlFor='search-bar'>
-            <input type='search' name='search-bar' className='Library-Search-Input' placeholder='Search Library Here...' value={searchInput} onChange={(e) => setSearchInput(e.target.value)} />
+            <input
+              type='search'
+              name='search-bar'
+              className='Library-Search-Input'
+              placeholder='Search Library Here...'
+              value={searchInput}
+              onChange={(e) => setSearchInput(e.target.value)}
+            />
           </label>
           {/* Genre Book selector dropdown that only appears on mobiles < 425px in screen width */}
           <div className='Library-Genre-Select-Dropdown'>
```

Comment:
Code looks great just can't seem to make a comment on the next file as its empty. 
src/components/main-carousel/MainCarousel.jsx - needs deleted as it was moved into the parent carousel folder. Parent folder /main-carousel can also be deleted.
src/components/main-carousel/shared-carousel-components/CarouselCards.jsx - Deleted aswell for the same reasoning.